### PR TITLE
Data Exporting

### PR DESCRIPTION
**Pull request by richraid21**
Currently only supports CSV. This feature has been mentioned in #59, #388 and #568 

Excerpt from the README (All properties are optional; the utility has it's own set of defaults) :    
```javascript
var csv = $("#grid").jsGrid("exportData", {
    type: "csv", //Only CSV supported
    subset: "all" | "visible", //Visible will only output the currently displayed page
    delimiter: "|", //If using csv, the character to seperate fields
    includeHeaders: true, //Include header row in output
    encapsulate: true, //Surround each field with qoutation marks; needed for some systems
    newline: "\r\n", //Newline character to use
    
    //Takes each item and returns true if it should be included in output.
    //Executed only on the records within the given subset above.
    filter: function(item){return true},
    
    //Transformations are a way to modify the display value of the output.
    //Provide a key of the field name, and a function that takes the current value.
    transformations: {
        "Married": function(value){
            if (value === true)
                return "Yes"
            if (value !== false)
                return "No"
        }
    }
});
```

The feature may need some tweaking, but I thought this was a good baseline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -934,6 +934,42 @@ $("#grid").jsGrid("loadData", { Name: "John" }).done(function() {
...
+    filter: function(item){return true},
+    
+    //Transformations are a way to modify the display value of the output.
+    //Provide a key of the field name, and a function that takes the current value.
+    transformations: {
+        "Married": function(value){
+            if (value === true)
+                return "Yes"
+            if (value !== false)
+                return "No"
```
Comment (Komap) [inline README.md]: if (value !== false)
 return "No"
wtf?
xD
realy o_O ???

Comment (Oxicode) [inline README.md]: lol

Comment (richraid21) [inline README.md]: I wrote the README when I was tired, sorry.

**Comment by tabalinas** [comment]
Thank you very much for the PR!

File: README.md (hunk 1 above)
Comment (Komap) [inline README.md]: No problem, guys, welcome from Ukraine :)

**Comment by Komap** [comment]
NP :) thank you too

**Comment by arirangz** [comment]
That's a great feature!

When do you plan to release it in the dist?

Thanks

**Comment by ansioncai** [comment]
I just wonder how can i get the last version that contain the 'exportData' feature

**Comment by Wardrop** [comment]
A new release seems overdue to include such new feature as this.

**Comment by klingamdinne** [comment]
Can i have a sample code how we can use exportData feature?

**Comment by Spucki** [comment]
How can I use the export function? Can someone please give me an example code?

**Comment by assadnazar** [comment]
Dev Team, any update?

**Pull request merged**